src/Meter/README.md

```diff
@@ -2,17 +2,33 @@
 
 A Meter is a simple UI component used to display a measurement (usually a percentage) on a known scale.
 
-## Props
-|Name|Type|Default|Description|
-|----|----|-------|-----------|
-| **value** | <code>Number</code> |  | **required**. Current value |
-| **min** | <code>Number</code> | <code>0</code> | *optional*. Minimum value |
-| **max** | <code>Number</code> | <code>100</code> | *optional*. Maximum value |
-| **labelFormatter** | <code>Function</code> | <code>"labelFormatter"</code> | *optional*. Function in which you can define a custom label format |
-| **ranges** | <code>Ranges</code> |  | *optional*. Array of Object in which you can define startValue, endValue, labelColor, fillingColor |
-| **baseLabelColor** | <code>String</code> |  | *optional*. Fallback labelColor |
-| **baseFillingColor** | <code>String</code> |  | *optional*. Fallback fillingColor |
-| **baseBackgroundColor** | <code>String</code> |  | *optional*. Fallback backgroundColor |
-| **id** | <code>String</code> |  | *optional*. Custom `id` for wrapper element |
-| **className** | <code>String</code> |  | *optional*. Additional `className` for wrapper element |
-| **style** | <code>Object</code> |  | *optional*. Inline-style overrides for wrapper element |
\ No newline at end of file
+![cover](https://user-images.githubusercontent.com/10867086/36546429-4d6179f2-17e3-11e8-8cc7-04d362304f7c.jpg)
+
+Typically, there are two use cases for this component:
+- display a value, such as a battery charge indicator
+- display the progress of an action
+
+## display a value
+
+In this use case, the value can go up and down based on the semantics of the real world object.
+
+![displayavalue](https://user-images.githubusercontent.com/10867086/36546866-5c9e3efe-17e4-11e8-9b7a-4ede95f0faa3.jpg)
+
+For example, the battery charge goes up while recharging, and down while using the phone. Regarding ranges, there might be a "low battery" range in red between 0% and 15%, and a "normal" battery range between 15% and 100%.
+
+If the meter is displaying the space occupied, the semantics are inverted: it will become red as it approaches 100% or the maximum space value in Gigabytes.
+
+## display the progress
+
+In this use case, the value starts from 0 and can only go up, typically to 100%.
+
+![progressbar](https://user-images.githubusercontent.com/10867086/36546608-c77eb2fe-17e3-11e8-9f72-5bac98150877.jpg)
+
+Follow these high-level rules:
+- do not show a progress indicator for actions that typically require less than a second
+- show a simple indicator (no label) for actions that typically require less than 10 seconds
```

Comment:
@lucacioria can you make an example? what's the difference between this and the following one?
(maybe make an example below as well)